gpu-operator/release-notes.rst

```diff
@@ -123,7 +123,29 @@ Fixed Issues
 Known Limitations
 ------------------
 
+* NVIDIA vGPU Manager does not work correctly on nodes with GPUs that require Open Kernel module drivers and GPU System Processor (GSP) firmware.
+  The logs for vGPU Manager pods include lines like the following example:
```

Comment:
The logs are actually from the `vGPU Device Manager` pods (I know it is confusing).